py-polars/polars/dataframe/plotting.py

```diff
@@ -35,6 +35,73 @@
     ]
 
 
+def configure_chart(
+    chart: alt.Chart,
+    *,
+    title: str | None,
+    x_axis_title: str | None,
+    y_axis_title: str | None,
+) -> alt.Chart:
+    """
+    A nice-looking default configuration, produced by Altair maintainer.
+
+    Source: https://gist.github.com/binste/b4042fa76a89d72d45cbbb9355ec6906.
+    """
+    properties = {}
+    if title is not None:
+        properties["title"] = title
+    if x_axis_title is not None:
+        chart.encoding.x.title = x_axis_title
+    if y_axis_title is not None:
+        chart.encoding.y.title = y_axis_title
+    return (
+        chart.properties(**properties)
+        .configure_axis(
+            labelFontSize=16,
+            titleFontSize=16,
+            titleFontWeight="normal",
+            gridColor="lightGray",
+            labelAngle=0,
+            labelFlush=False,
+            labelPadding=5,
+        )
+        .configure_axisY(
+            domain=False,
+            ticks=False,
+            labelPadding=10,
+            titleAngle=0,
+            titleY=-20,
+            titleAlign="left",
+            titlePadding=0,
```

Comment:
```suggestion
            # The settings below move the axis title on top of the axis
            # and orient it horizontally
            titleAngle=0,
            titleAlign="left",
            titlePadding=0,
            # This pushes the title high enough so it does not overlap with the latest tick.
            # For some edge cases, this might still happen and there is no automated way
            # to resolve it right now but it usually works well.
            titleY=-25,
```